ui/src/pages/Tags/Detail/index.tsx

```diff
@@ -145,9 +145,6 @@ const Index: FC = () => {
 
             <p className="text-break">
```

Comment:
Looking at the latest requirements, you need to parse html here

<div
    className="text-break"
    dangerouslySetInnerHTML={{ __html: tagInfo.excerpt }}
  />